instance/settings-sample.py

```diff
@@ -62,3 +62,10 @@
 SUPPORT_EMAIL = 'person@example.com'
 #: Sitemap key
 SITEMAP_KEY = None
+DOGPILE_CACHE_URLS = "http://127.0.0.1:6379"
+DOGPILE_CACHE_REGIONS = [
+    ('hour', 3600),
+    ('day', 3600 * 24),
+    ('month', 3600 * 24 * 31),
```

Comment:
I recommend not mixing up region names and time periods. Give these distinct labels such as `index`.